prediction_market_agent/agents/safe_guard_agent/guards/goplus_.py

```
@@ -243,20 +289,20 @@ def _build_validation_result(malicious_reasons: list[str]) -> ValidationResult:
-def _goplus_call(f: Callable[[], T], retry: bool = True) -> T | None:
+def _goplus_call(f: Callable[[], T], retries: int = 3) -> T | None:
     response = f()
     code = int(response.code)  # type: ignore # GoPlus isn't typed.
 
     # See https://docs.gopluslabs.io/reference/api-status-code for possible status codes.
     if code == 1:
         return response
 
-    elif code in (2, 4029) and retry:
+    elif code in (2, 4029) and retries > 0:
         logger.warning(f"Goplus rate limit hit, waiting a bit and trying again.")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)